windows/tools/nocli/Main.cxx

```diff
@@ -59,15 +59,25 @@ CtrlHandler(DWORD controlType)
     return FALSE;
 }
 
+/**
+ * @brief Instance IDs for PLOG loggers
+ */
+enum class LogInstanceId {
+    // Default logger is 0 and is omitted from this enum
+    Console = 1,
+    File = 2,
+};
+
 int
 main(int argc, char* argv[])
 try {
-    plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("nocli").c_str());
-    plog::DebugOutputAppender<plog::TxtFormatter> debugAppender;
-    plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
-    plog::init(plog::verbose, &rollingFileAppender)
-        .addAppender(&debugAppender)
-        .addAppender(&colorConsoleAppender);
+    static plog::ColorConsoleAppender<plog::MessageOnlyFormatter> colorConsoleAppender{};
+    plog::init<static_cast<int>(LogInstanceId::Console)>(plog::info, &colorConsoleAppender);
+
+    static plog::RollingFileAppender<plog::TxtFormatter> rollingFileAppender(logging::GetLogName("nocli").c_str());

```

Comment:
Why are these `static`?

Reply:
Oh I just noticed that that's what the PLOG github examples use, but they don't need to be

Reply:
From the github:
<img width="411" alt="image" src="https://github.com/microsoft/nearobject-framework/assets/43283510/1e3c1b88-22db-40a0-b729-6d55f7c127c8">